### DISCUSSION TITLE
Loading Data Sets Into Local Machine For Review

### DISCUSSION THREAD

**Discussion by AxiomInfinite** [telekom-security/tpotce#1085] [Q&A]
Hi everyone,
I'm relatively new to the honeypot and have about 8 days of data and currently still running in an AWS environment. Would i be able to save/grab all the data from AWS instance and load this locally and have access to the visual set up this gives? How would I do something like that as I feel like I'm very close and just missing something. Currently have it running localy on my own hardware, but trying to figure out how to just kill the honeypot and load log files to walk through it all in the dashboard.

**Comment by lcia-projects** [comment]
i think the best approach would be a hive sensor install in AWS and have a hive server at your house. data would be recorded in the cloud then sent to your home server for analysis and long term storage.
